transform/mattermost-analytics/models/staging/mm_telemetry_prod/stg_mm_telemetry_prod__activity.sql

```diff
@@ -0,0 +1,73 @@
+
+
+with source as (
+    select * from {{ source('mm_telemetry_prod', 'activity') }}
+),
+renamed as (
+
+    select
+        -- Common event columns
+        id               as event_id
+        , event          as event_table
+        , event_text     as event_name
+        , user_id        as server_id
+        , received_at    as received_at
+        , timestamp      as timestamp
+
+        -- Server info
+        , coalesce(context_traits_installationid,  context_traits_installation_id) as installation_id
+        , anonymous_id
+        , context_ip as server_ip
+
+        -- Channel metrics
+        , public_channels as count_public_channels
+        , public_channels_deleted as count_deleted_public_channels
+        , private_channels as count_private_channels
+        , private_channels_deleted as count_deleted_private_channels
+        , bot_posts_previous_day as count_bot_posts_previous_day
+        , posts_previous_day as count_posts_previous_day
+        , teams as count_teams
+        , slash_commands as count_slash_commands
+        , direct_message_channels as count_direct_message_channels
+        , posts as count_posts
+        , incoming_webhooks as count_incoming_webhooks
+        , outgoing_webhooks as count_outgoing_webhooks
+
+        -- User & account metrics
+        , active_users_daily as daily_active_users
+        , active_users_monthly as monthly_active_user
+        , registered_users as count_registered_users
+        , registered_deactivated_users as count_registered_deactivated_users
+        , bot_accounts as bot_accounts
+        , guest_accounts as guest_accounts
+
+        -- Metadata from Rudderstack
+        , context_library_version
+        , context_library_name
+        , uuid_ts
```

Comment:
maybe update the naming conventions in https://mattermost.atlassian.net/wiki/spaces/DATAENG/pages/2422603777/DWH+Naming+Conventions that `*_ts` is acceptable as well for timestamps - like `timestamp` (or rename this one)

Reply:
I was between keeping the value as is or completely dropping it. It's never used downstream. Also according to [Rudderstack's docs](https://www.rudderstack.com/docs/destinations/warehouse-destinations/warehouse-schema/) "Added by RudderStack for debugging purposes. Can be ignored for analytics."

Reply:
I think so, we can always add it necessary.